db.sql

```diff
@@ -2019,6 +2019,7 @@ CREATE TABLE `ticket_statuses` (
   `ticket_status_name` varchar(200) NOT NULL,
   `ticket_status_color` varchar(200) NOT NULL,
   `ticket_status_active` tinyint(1) NOT NULL DEFAULT 1,
+  `ticket_status_order` int(11),
```

Comment:
@johnnyq - Do we want the default for `ticket_status_order` to be 0 or NULL? Whatever we decide, it should be explicit.

Reply:
The default should be 0